control-plane/ctrl/client-protocols/tcp.go

```diff
@@ -58,13 +58,13 @@ func (tcp *TcpObj) StartDiscovery(jsonData []byte) *shared.DataObj {
 	for {
 		_, err := tcpConnObj.connSock.Write(jsonData)
 		if err != nil {
-			fmt.Printf("Cannot send!!! %s\n", err)
+			fmt.Printf("Discovery: Cannot send!!! %s\n", err)
 			os.Exit(1)
 		}
 
 		bytes, err := tcpConnObj.connSock.Read(buf)
 		if err != nil {
-			fmt.Printf("Cannot read!!!! msg: %s\n", err)
+			fmt.Printf("Discovery: Cannot read!!!! msg: %s\n", err)
```

Comment:
Not this unprofessional thousand of exclamation marks!!!!!!!! I don't know any professional program printing any of this signs ... Please remove and write a helpful message, if any.